src/main/java/appeng/services/version/ModVersionFetcher.java

```diff
@@ -30,11 +53,18 @@ public Version get()
 		{
 			return new DoNotCheckVersion();
 		}
-		else
+
+		try
 		{
 			final Version version = this.parser.parse( this.rawModVersion );
 
 			return version;
 		}
+		catch( final Exception e )
```

Comment:
should be more specified, what kind of exception can even be thrown here. The only one I did add, was an `AssertionError`